### DISCUSSION TITLE
Your opinion

### DISCUSSION THREAD

**Discussion by syn-4ck** [syn-4ck/pynipper-ng#38] [General]
What do you think about pynipper-ng? Give me feedback 👍

**Comment by Coldfirex** [comment]
I think this is a fascinating project.  Are you still working on it?

**Reply by syn-4ck** [reply]
Hi,

My idea is work on it and maintain the project, but in this moment I don't have enought time to develop it.

The roadmap is try to build a solution with all the CIS rules for relevant devices, but do this requires a lot of development time.

Regards.
